### PR TITLE
Added message overflow checking

### PR DESCRIPTION
**Pull request by ppajersk**
Cidrbot now splits messages up if they exceed a certain limit

### CODE REVIEW

File: wxt_cidrbot/cidrbot.py [hunk 1]
```diff
@@ -166,17 +169,49 @@ def webhook_request(self, event):
             if org_id == self.orgID:
                 self.message_event(json_string, event_type, webex_msg_sender)
 
+    def check_message_overflow(self, message, room_id, message_id, pt_id, request_type):
+        split_keyword = 'Repo:'
+
+        if len(message) < 4800:
```
Comment (jlothian) [inline wxt_cidrbot/cidrbot.py:175]: best to make this value a constant somewhere, maybe self.MAX_MESSAGE_LENGTH or something

File: wxt_cidrbot/cidrbot.py [hunk 2]
```diff
@@ -166,17 +169,49 @@ def webhook_request(self, event):
...
+                    if '**All Issues:**' not in repo:
+                        message_first_part += split_keyword
+                    message_first_part += repo
+                elif len(remainder_message) + len(repo) < 4800:
+                    remainder_message += "Repo:" + repo
+                else:
+                    remainder_message = "One repo has too many issues and exceeds the webex message limit."
+
+        if request_type == 'daily_message':
+            self.send_wbx_msg(room_id, message_first_part, pt_id)
```
Comment (jlothian) [inline wxt_cidrbot/cidrbot.py:194]: does it make sense to simply this block and pull the send_wbx_msg outside of the ifs?

**Review by jlothian — CHANGES_REQUESTED** [review]

File: wxt_cidrbot/cidrbot.py (hunk 1 above)
Comment (ppajersk) [reply wxt_cidrbot/cidrbot.py]: Changed to self.webex_message_char_limit

File: wxt_cidrbot/cidrbot.py (hunk 2 above)
Comment (ppajersk) [reply wxt_cidrbot/cidrbot.py:195]: Moved self.send_wbx_msg() outside of block, unnecessary duplication of line, since it runs in both cases

**Review by jlothian — APPROVED** [review]

**Pull request merged**